### PR TITLE
fix: accountType doc

### PR DESCRIPTION
**Pull request by flochaz**
Fixing #112

### CODE REVIEW

File: source/aws-bootstrap-kit/lib/account.ts [hunk 1]
```diff
@@ -69,10 +69,21 @@ export interface IAccountProps {
...
+   * The account used to deploy CI/CD pipelines (See [here](https://cs.github.com/?scopeName=bk&scope=repo%3Aawslabs%2Faws-bootstrap-kit&q=AccountType.CICD) for internal usage).
+   */
   CICD = "CICD",
-  DNS = "DNS",
+  /**
+   * Accounts which will be used to deploy Stage environments (staging/prod ...). (See [here](https://cs.github.com/?scopeName=bk&scope=repo%3Aawslabs%2Faws-bootstrap-kit&q=AccountType.STAGE) for internal usage)
+  **/
   STAGE = "STAGE",
+  /**
+   * Accounts dedicated to developpers work.
```
Comment (moltar) [inline source/aws-bootstrap-kit/lib/account.ts:85]: Aka "sandbox"?

https://aws.amazon.com/blogs/mt/best-practices-creating-managing-sandbox-accounts-aws/

Comment (moltar) [inline source/aws-bootstrap-kit/lib/account.ts:85]:
```suggestion
   * Accounts dedicated to developers work. 
```

File: source/aws-bootstrap-kit/lib/account.ts [hunk 2]
```diff
@@ -97,6 +108,12 @@ export class Account extends Construct {
 
     const accountProvider = AccountProvider.getOrCreate(this);
 
+    const children = this.node.findAll();
```
Comment (moltar) [inline source/aws-bootstrap-kit/lib/account.ts:111]: Seems like an unrelated change. Is this deliberate?

File: source/aws-bootstrap-kit/lib/account.ts [hunk 3]
```diff
@@ -69,10 +69,21 @@ export interface IAccountProps {
...
+ * The type of the AWS account 
+ **/
 export enum AccountType {
+  /**
+   * The account used to deploy CI/CD pipelines (See [here](https://cs.github.com/?scopeName=bk&scope=repo%3Aawslabs%2Faws-bootstrap-kit&q=AccountType.CICD) for internal usage).
+   */
   CICD = "CICD",
-  DNS = "DNS",
+  /**
+   * Accounts which will be used to deploy Stage environments (staging/prod ...). (See [here](https://cs.github.com/?scopeName=bk&scope=repo%3Aawslabs%2Faws-bootstrap-kit&q=AccountType.STAGE) for internal usage)
```
[the remark this answers is not in the document]
Comment (flochaz) [reply source/aws-bootstrap-kit/lib/account.ts:81]: In order to define an environment where workloads could. be deployed, I think STAGE make more sense and relates better to CI/CD way of defining things.

File: source/aws-bootstrap-kit/lib/account.ts (hunk 1 above)
Comment (flochaz) [reply source/aws-bootstrap-kit/lib/account.ts]: indeed SANDBOX works as well , will mention it to the doc
